django/db/backends/base/references.py

```diff
@@ -19,6 +19,16 @@ def references_column(self, table, column):
         """
         return False
 
+    def rename_table_references(self, old_table, new_table):
+        """
+        Rename all references to the old_name to the new_table.
+        """
+
```

Comment:
Looks a bit odd without any content -- not sure if I've seen that elsewhere in Django. Maybe put a `pass` there?